modules/apps/comment/comment-taglib/src/main/java/com/liferay/comment/taglib/internal/context/DefaultCommentTreeDisplayContext.java

```diff
@@ -136,6 +147,35 @@ public boolean isEditControlsVisible() throws PortalException {
 		return hasUpdatePermission();
 	}
 
+	public boolean isEditDeletePermission() throws PortalException {
+		long groupId = _discussionRequestHelper.getScopeGroupId();
+
+		CommentGroupServiceConfiguration commentGroupServiceConfiguration =
+			_getCommentGroupServiceConfiguration(groupId);
+
+		boolean editComment =
+			commentGroupServiceConfiguration.
+				enableOwnersToEditOtherUserComments();
+
+		boolean deleteComment =
+			commentGroupServiceConfiguration.
+				enableOwnersToDeleteOtherUserComments();
+
+		if (isGroupAdmin()) {
+			return true;
+		}
+		else if (editComment || deleteComment) {
+			return true;
+		}
+		else {
```

Comment:
could this be van "else if" clause as well?
the final statement of the method would then be "return false;" outside of conditional branches